.github/workflows/publish.yaml

```diff
@@ -0,0 +1,20 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registries:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Run docker build
+        run: make docker-build
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: vulcanizejenkins
+          password: ${{ secrets.VULCANIZEJENKINS_PAT }}
+          repository: vulcanizejenkins/eth-header-sync
```

Comment:
The repo should be vulcanize/eth-header-sync

Reply:
@AFDudley In Docker Hub our name is vulcanizejenkins. Could you give access to `vulcanize` account?

Reply:
vulcanizejenkins already has access to the vulcanize account, please test to confirm.